.github/workflows/build-docs.yml

```
@@ -0,0 +1,30 @@
+name: "Documentation Pull Request Docs Check"
+on:
+  pull_request:
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+        
+      - name: Install prerequisites
+        run: |
+          sudo apt-get --allow-releaseinfo-change update -y
+          sudo apt-get install -y tox pandoc
+          
+      - name: Building Documentation in html
+        run: |
+          echo "---------------------------------------------------------------------------------------"
+          echo "---------------------------------- Building html --------------------------------------"
+          echo "---------------------------------------------------------------------------------------"
+          tox -e docs
+          echo "Building html was successful!"
+          
+      - name: Store html build result
+        uses: actions/upload-artifact@v3

```

actions/upload-artifact@v3 will be deprecated. 

```suggestion
        uses: actions/upload-artifact@v4

```